CHANGELOG.md

```diff
@@ -5,20 +5,65 @@ All notable changes to Zebra are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org).
 
 
-## Next Release (Draft)
+## [Zebra 1.0.0-beta.13](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-beta.13) - 2022-07-11
 
-This release improves Zebra's sync and verification performance under heavy load.
-(TODO - complete the summary.)
+This release fixes multiple bugs in proof and signature verification, which were causing big performance issues near the blockchain tip.
+It also improves Zebra's sync performance and reliability under heavy load.
+
+### Disk and Network Usage Changes
+
+Zebra now uses around 50 - 100 GB of disk space, because many large transactions were recently added to the block chain.
```

Comment:
Copying Daira's suggestions here as well:

```suggestion
Zebra now uses around 50 - 100 GB of disk space, because many large transactions were recently added to the block chain. (In the longer term, several hundred GB are likely to be needed.)
```